main.go

```diff
@@ -103,101 +102,50 @@ func runMemphis(s *server.Server) {
 		s.Errorf("Failed initializing analytics: " + err.Error())
 	}
 
-	err = server.TenantSeqInitialize()
-	if err != nil {
-		s.Errorf("Failed to initialize tenants sequence %v", err.Error())
-	}
-
-	err = memphis_cache.InitializeUserCache(s.Errorf)
-	if err != nil {
-		s.Errorf("Failed to initialize user cache %v", err.Error())
-	}
-
-	err = s.InitializeEventCounter()
-	if err != nil {
-		s.Errorf("Failed initializing event counter: " + err.Error())
-	}
-
-	err = s.InitializeFirestore()
-	if err != nil {
-		s.Errorf("Failed initializing firestore: " + err.Error())
-	}
-
-	s.InitializeMemphisHandlers()
-
-	err = server.InitializeIntegrations()
-	if err != nil {
-		s.Errorf("Failed initializing integrations: " + err.Error())
-	}
-
-	err = s.SetDlsRetentionForExistTenants()
-	if err != nil {
-		s.Errorf("failed setting existing tenants with dls retention opts: %v", err.Error())
-	}
+	go http_server.InitializeHttpServer(s)
```

Comment:
Move this one right before the the printings so that the UI will be avilable only when the server done its initialization